src/uct/api/uct_def.h

```diff
@@ -309,6 +315,11 @@ typedef struct uct_cm_listener_conn_request_args {
      * Remote data from the client.
      */
     const uct_cm_remote_data_t *remote_data;
+
+    /**
+     * Client's address.
+     */
+    ucs_sock_addr_t             client_address;
```

Comment:
@alinask - all this args are [in] for the callback, correct ?

Reply:
> @alinask - all this args are [in] for the callback, correct ?

@shamisp these are fields in the uct_cm_listener_conn_request_args_t struct which is one of the [in] params for the callback - https://github.com/openucx/ucx/blob/master/src/uct/api/uct_def.h#L582